WRITING.md

```diff
@@ -1,238 +1,127 @@
-- Reference pages should link to concept explainers
-- Prerequisite knowledge should be disclosed (at the top of a page, list what the reader needs to know, and link to the appropriate pages).
-- At the end of a tutorial, provide next steps for learning
-- Link to external information, like Ryan's streams, that can further learning beyond the docs
-
-</details>
-
-## Content types
-
-We're using [Diataxis](https://diataxis.fr/) as a starting point for our docs. Their [needs](https://diataxis.fr/needs/) page is a must-read introduction!
-
-The key insight of Diataxis is that documentation requires different _kinds_ of writing to meet the various needs of learners. A detailed API reference won't serve someone who's starting from scratch. A project-based beginner tutorial is great for onboarding new users holistically, but not great for "filling in the gaps" of someone's knowledge.
-
-| Diataxis Content Type                               | What We Call Them                       | Focus                                                                                                               |
-| --------------------------------------------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
-| [Tutorials](https://diataxis.fr/tutorials/)         | (Project-based / interactive) Tutorials | To _introduce, educate_. Get the reader up and running and comfortable using Solid.                                 |
-| [How-to Guides](https://diataxis.fr/how-to-guides/) | How-to Guides                           | To _guide, demonstrate_. Show how to accomplish a task or a set of tasks.                                           |
-| [Reference](https://diataxis.fr/reference/)         | API Reference                           | To _describe, inform_. Provide a comprehensive description of Solid's API.                                          |
-| [Explanation](https://diataxis.fr/explanation/)     | Concept Guides, Diving Deeper Guides    | To _clarify, discuss_. Allow the reader to understand the concepts at play; explain the behind-the-scenes of Solid. |
-
-![image](https://user-images.githubusercontent.com/4033662/174461439-01f1c0f6-687d-4807-ac43-f61654144886.png)
-
-## Tone and Style
-
-_This section is especially looking for feedback_
-
-Our first instinct for a tone guide was simply:
-
-```
-Friendly tone! Approachable is better than technical
-```
-
-This begs the question: what is a "friendly tone," exactly? It's hard to codify this feeling, but [Google's voice and tone](https://developers.google.com/style/tone) guide does a great job, and we defer to that guide.
+Modern documentation has shifted toward a less formal tone to great effect.
+Good documents these days center on providing a friendly and approachable tone.
+Solid would like to adopt this.
```

Comment:
[feat] I provided some examples of how contributors can make their writing more accessible to users.  
```suggestion
- Use gender-neutral language 
Ex: Solid is focused on making the developer's experience just as good and predictable as his user's experience, and here are some ways in which we are able to achieve that. ----> Solid is focused on making the developer's experience just as good and predictable as their users' experience, and here are some ways in which we are able to achieve that.
- Avoid using terms that have racist undertones
Ex: Make sure to push your contributions to the  `master` branch. ---> Make sure to push your contributions to the `main` branch. 
```